Hetic-web3-g1/PROJET_FIL_ROUGE_W3_G4
# Searchbar features

Presentation:
- Add click outside / debounce time to searchbar

How to test:
- Test to search to display panel then click outside, the block should be hidden
